File: test/java/test_CropHeatFlux.java
//Test generation'

import java.math.BigDecimal;
public static class Test
{
    //first);
    //test1);

    public static void test1()
    {
        double netRadiationEquivalentEvaporation = 638.142;
        double soilHeatFlux = 188.817;
        double potentialTranspiration =  1.413;

        CropHeatFlux res0 = Estimation_CropHeatFlux.CalculateCropHeatFlux(netRadiationEquivalentEvaporation,soilHeatFlux,potentialTranspiration);
        System.out.println(" cropHeatFlux: "+res0.cropHeatFlux);

        System.out.println(((new BigDecimal(res0.cropHeatFlux)).setScale(3, BigDecimal.ROUND_HALF_DOWN)).equals((new BigDecimal( 447.912)).setScale(3, BigDecimal.ROUND_HALF_DOWN)));
    }
}

File: test/cs/test_CropHeatFlux.cs
'Test generation'

public static class Test
{
    //first);
    //test1

    public static void test1()
    {
        CropHeatFlux res0 = Estimation_CropHeatFlux.CalculateCropHeatFlux(soilHeatFlux:188.817,potentialTranspiration: 1.413,netRadiationEquivalentEvaporation:638.142);

        Console.WriteLine("{0}","cropHeatFlux: "+res0.cropHeatFlux);

        Console.WriteLine("cropHeatFlux Comparison: ("+Math.Round( 447.912,3)+";"+Math.Round(res0.cropHeatFlux,3)+") "+Equals(Math.Round( 447.912,3),Math.Round(res0.cropHeatFlux,3)));
    }
}
